website/frontend/src/pages/OurProducts/CalibrationPage.js

```diff
@@ -0,0 +1,99 @@
+
+const CalibrationPage = () => {
+  useInitScrollTop();
+  return (
+    <Page>
+      <div className="product-page calibrate mobile-app analytics api">
+        <Header
+          style = {{ backgroundColor: '#e6e8fa' }}
+          pageTitle={'AirQalibrate'}
+          title={'Calibrate your low-cost sensor data'}
+          subText={
+            'Improve the accuracy and reliability of data from your low-cost air quality monitors using AirQo’s AirQalibrate tool, delivering more accurate results.'
+          }>
+          <img src={HeaderImage} alt="" style={{ borderRadius: '8px'}} />
+        </Header>
+        <div className="content">
+          <div className="grid-wrapper column section section-1">
+            <div className="row">
+              <h2 className="left title">
+                <span className="blue"> Why calibrate</span> your low-cost sensor data?
+              </h2>
+              <p className="right">
+              Low cost sensors are relied on to increase the geographical coverage of air quality monitoring networks. 
+              However, they are sensitive to ambient conditions such as humidity and temperature which could affect 
+              the accuracy of air quality data and require field calibration to improve their performance.              
+              Calibration enhances the quality and reliability of air quality data from low-cost sensors
+              </p>
+            </div>
+            <div className="overlap-section">
+              <div className="lapping-left card" /*card-sm larger-top*/  style={{ backgroundColor: '#FFFCE1' }}>
+                <h5>Cost effective and accessible</h5>
+                <p>
+                AirQalibrate is a Machine Learning based calibration tool that eliminates the need for reference grade monitor 
+                or on-site monitor calibration.<br />
+                <br />
+                AirQalibrate enables users without access to reference grade monitors or technical expertise 
+                to calibrate data from their low-cost monitors and improve performance and build trust in the air quality network.<br />
+                <br />
+                It reduces operational costs of monitoring thereby enabling the expansion of monitoring systems 
+                to create a dense air quality monitoring and management network.
+                </p>
+              </div>
+              <div className="lapping-left image" id="section-1">
+                <img src={Section1Image} alt="" />
+              </div>
+            </div>
+          </div>
+
+          <div className="grid-wrapper section">
+            <div className="overlap-section">
+              <div className="lapping-right card" style={{ backgroundColor: '#E6E6FA', marginTop: '30px' }}>
+                <h5>Calibrate your data</h5>
+                <p>
+                Upload a CSV containing your low-cost sensor  PM<sub>2.5</sub> and PM<sub>10</sub> 
```

Comment:
- Change "a CSV containing" to " a CSV file containing". 
- Add a space between the words PM10 and data
- Add a comma after PM10 data
- Change "follow simple steps" to "follow a few simple steps,"